### ISSUE TITLE
LSP Typescript | Go Definition Error

### ISSUE THREAD

**Issue by jamszh** [jamesz96/dotfiles#12]
When I try to use LSP `go-definition` for the first time to a typescript declaration file (`app.d.ts`):
Works on subsequent attempts.

Probably lazy loading related

🤔 

Error log
```
[C]: in function '__index'
        /usr/local/share/nvim/runtime/lua/vim/lsp.lua:1092: in function 'set_defaults'
        /usr/local/share/nvim/runtime/lua/vim/lsp.lua:1515: in function '_on_attach'
        /usr/local/share/nvim/runtime/lua/vim/lsp.lua:1663: in function 'buf_attach_client'
        ...al/share/nvim/lazy/nvim-lspconfig/lua/lspconfig/util.lua:390: in function 'add'
        ...share/nvim/lazy/nvim-lspconfig/lua/lspconfig/configs.lua:254: in function 'try_add'
        ...share/nvim/lazy/nvim-lspconfig/lua/lspconfig/configs.lua:76: in function <...share/nvim/lazy/nvim-lspconfig/lua/lspconfig/configs.lua:75>
        [C]: in function 'nvim_cmd'
        ...ocal/Cellar/neovim/0.6.1/share/nvim/runtime/filetype.lua:23: in function <...ocal/Cellar/neovim/0.6.1/share/nvim/runtime/filetype.lua:22>
        [C]: in function 'nvim_buf_call'
        ...ocal/Cellar/neovim/0.6.1/share/nvim/runtime/filetype.lua:22: in function <...ocal/Cellar/neovim/0.6.1/share/nvim/runtime/filetype.lua:11>
        [C]: in function 'nvim_set_current_buf'
        /usr/local/share/nvim/runtime/lua/vim/lsp/util.lua:1092: in function 'jump_to_location'
        ...nvim/lazy/telescope.nvim/lua/telescope/builtin/__lsp.lua:194: in function 'handler'
        /usr/local/share/nvim/runtime/lua/vim/lsp.lua:1383: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

**Comment by jamszh** [comment]
Oh my god. It turns out the issue was coming from a repo specific configuration for `vscode`. That's right `vscode` not `neovim`.

Specifically this line

```json
"editor.defaultFormatter" : null
```


in file: `.vscode/settings.json`

I do not get this issue when I remove this line

**Comment by jamszh** [comment]
Fixed in #23
